importexport/musicxml/importmxmlpass2.h

```diff
@@ -130,20 +130,30 @@ class MusicXMLParserLyric {
       };
 
 //---------------------------------------------------------
-//   Notation
+//   Generic Annotation
+//      Most with text base attributes (font-*, placement)
 //---------------------------------------------------------
 
-class Notation {
+class GenericAnnotation {
 public:
-      Notation(const QString& name) { _name = name; }
+      GenericAnnotation(const QString& name, const QString& parent = "",
```

Comment:
I prefer to have this rename undone:
- the qualifier "generic" adds nothing here but additional characters (this is not a base class, there are no more specific annotations
- this class stores notations (which includes but is not limited to annotations), thus the name is misleading
- in the cpp code there is now a mix between the annotation and notation concepts